cpp/benchmarks/join/error.cuh

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#ifndef __ERROR_CUH
+#define __ERROR_CUH
+
+#include <cstdio>
+#include <cstdlib>
+#include <rmm/rmm.h>
+#include <cudf/cudf.h>
+
+
+#ifndef CUDA_RT_CALL
```

Comment:
These error utils shouldn't be use. Use what's in `src/utililites/error_utils.hpp`

Reply:
Resolved in https://github.com/rapidsai/cudf/pull/2105/commits/e9bd79fc86edb8560d4585975b743475fcd269cd.